src/test/suite/fold.test.ts

```diff
@@ -0,0 +1,74 @@
+import * as assert from "assert";
+import * as vscode from "vscode";
+import * as sinon from "sinon";
+import { getCursorlessApi } from "../../util/getExtensionApi";
+import { openNewEditor } from "../openNewEditor";
+
+suite("fold", async function () {
+  this.timeout("100s");
+  this.retries(3);
+
+  teardown(() => {
+    sinon.restore();
+  });
+
+  test("fold made", foldMade);
+  test("unfold made", unfoldMade);
+});
+
+async function foldMade() {
+  const graph = (await getCursorlessApi()).graph!;
+  const editor = await openNewEditor("function myFunk() {\n\n}", "typescript");
+  await graph.hatTokenMap.addDecorations();
+
+  await vscode.commands.executeCommand(
+    "cursorless.command",
+    "fold made",
+    "fold",
+    [
+      {
+        type: "primitive",
+        mark: {
+          type: "decoratedSymbol",
+          symbolColor: "default",
+          character: "m",
+        },
```

Comment:
The problem with using a mark here is that this test will break if we change our hat allocation algorithm, whereas for the recorded tests we capture the required allocations.  Maybe the easiest thing would be to just change this test to use the current selection, like we do for some of our [other manual tests](https://github.com/cursorless-dev/cursorless-vscode/blob/30bc0f2ad9ed0d070a256965913afb1e1a675557/src/test/suite/backwardCompatibility.test.ts#L23)?

Reply:
done